Gruntfile.js

```diff
@@ -92,13 +92,13 @@ grunt.initConfig({
 			}
 		},
 		qunit: [
-			"test/index.html",
```

Comment:
Can you move this file back? I very much like to stick with opening `qunit/test` to have tests run...